src/PixiEditor/PixiEditor.csproj

```
@@ -422,6 +422,8 @@
 		<Resource Include="Images\LanguageFlags\ru.png" />
 		<None Remove="Images\LanguageFlags\cs.png" />
 		<Resource Include="Images\LanguageFlags\cs.png" />
+    <None Remove="Images\LanguageFlags\tr.png" />
+    <Resource Include="Images\LanguageFlags\tr.png" />
```

Let's keep the indentation 

```suggestion
		<None Remove="Images\LanguageFlags\tr.png" />
		<Resource Include="Images\LanguageFlags\tr.png" />
```

---

I couldn't understand this part everything looks fine
[image](https://github.com/PixiEditor/PixiEditor/assets/25502590/bfeafd5b-5ab3-4204-9860-fe01447d5e5d)

---

Okay ı solved :)